modules/ossm-security-cert-manage.adoc

```diff
@@ -44,96 +44,63 @@ spec:
       istiod:
         type: PrivateKey
         privateKey:
-          rootCADir:  /etc/cacerts
+          rootCADir: /etc/cacerts
 ----
 
+[NOTE]
+====
+After creating/changing/deleting the `cacert` secret, the control plane `istiod` and `gateway` pods must be restarted so the changes go into effect. Use the following command to restart the pods: `oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway)'`.
+
+Restart the application pods so that the sidecar containers pick up the changes as well.
+====
+
 [id="ossm-cert-manage-verify-cert_{context}"]
 == Verifying your certificates
 
-Use the Bookinfo sample application to verify your certificates are mounted correctly. First, retrieve the mounted certificates. Then, verify the certificates mounted on the pod.
+Use the xref:../../service_mesh/v2x/ossm-create-mesh.adoc#ossm-tutorial-bookinfo-overview_ossm-create-mesh[Bookinfo example application] to verify that the workload certificates are signed by the certificates that were plugged into the CA. This requires you have `openssl` installed on your machine
```

Comment:
We can't put cross-references in modules (yet).  So this link needs to revert back to text.

Reply:
@JStickler I see this link work in my local build, so I must not understand what you mean.

Reply:
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references 
"Only use xref in assemblies, not modules."
Technically it works, but right now the rule is not to put XREFs in modules.

Reply:
Should we also include the istio-egressgateway pod in line 53 ?
`oc -n istio-system delete pods -l 'app in (istiod,istio-ingressgateway, istio-egressgateway)'`.